ocaml/dream/server.opam

```diff
@@ -0,0 +1,7 @@
+opam-version: "2.0"
+
+depends: [
+  "ocaml" {>= "5.2.0"}
+  "dream"
```

Comment:
Could you specify a version q@pyx ?

Reply:
Done, I pinned OCaml to 5.2.0 and dream to 1.0.0~alpha6, both are official (sort of in dream's case) stable releases.